src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,158 @@
+$base-margin: 0;
+$base-font: roboto, helvetica, sans-serif;
+$text-color: #616070;
+$blue-color: #00acdc;
+$transition-duration: 0.3s;
+
+.card-container {
+  display: grid;
+  row-gap: 46px;
+  column-gap: 48px;
+  padding: 50px 40px;
+  justify-content: center;
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, 200px);
+  }
+
+  @media (min-width: 768px) and (max-width: 1023px) {
+    grid-template-columns: repeat(3, 200px);
+  }
+
+  @media (min-width: 488px) and (max-width: 767px) {
+    grid-template-columns: repeat(2, 200px);
+  }
+
+  @media (max-width: 487px) {
+    grid-template-columns: repeat(1, 200px);
+  }
+}
+
+.card {
+  flex-direction: column;
+  display: flex;
+  box-sizing: border-box;
```

Comment:
move ` box-sizing` into `*` selector

Reply:
after this change everything is located not like on the reference at all